bundle-kubeflow-lite.yaml

```
@@ -0,0 +1,48 @@
+bundle: kubernetes
+name: kubeflow-lite
+applications:
+  admission-webhook:             { charm: admission-webhook,       channel: latest/edge, scale: 1 }
+  argo-controller:               { charm: argo-controller,         channel: latest/edge, scale: 1 }
+  dex-auth:                      { charm: dex-auth,                channel: 2.28/edge, scale: 1, trust: true }
```

Restore accidentally deleted envoy charm
```suggestion
  dex-auth:                      { charm: dex-auth,                channel: 2.28/edge, scale: 1, trust: true }
  envoy: { charm: envoy, channel: latest/edge, scale: 1 }
```